libraries/AP_Motors/AP_MotorsMatrix_6DoF_Scripting.cpp

```diff
@@ -0,0 +1,327 @@
+/*
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifdef ENABLE_SCRIPTING
+
+#include <AP_HAL/AP_HAL.h>
+#include "AP_MotorsMatrix_6DoF_Scripting.h"
+#include <AP_Vehicle/AP_Vehicle.h>
+
+extern const AP_HAL::HAL& hal;
+
+void AP_MotorsMatrix_6DoF_Scripting::output_to_motors()
+{
+    int8_t i;
```

Comment:
you can probably make this a uint8_t and move it down into the for loop line

Reply:
done